scss/_patterns_navigation.scss

```diff
@@ -208,6 +214,10 @@ $spv-navigation-logo: 0.3125rem;
       position: absolute;
       top: 0;
       width: $navigation-logo-tag-width;
+
+      @media (max-width: $breakpoint-navigation-threshold) {
```

Comment:
Ideally we should do this other way around (mobile first), so by default smaller spacing, and `min-width` breakpoint to turn it into desktop view.